### ISSUE TITLE
Masks from transductive-voc make no sense.

### ISSUE THREAD

**Issue by dennisushi** [wenbowen123/BundleTrack#68]
Hi,

I run it for the mustard_easy_00_02 scene, with the provited init frame  `python transductive-vos.pytorch/run_video.py --img_dir ./data/mustard_easy_00_02/rgb/ --init_mask_file ./data/mustard_easy_00_02/init_mask.png --mask_save_dir ./data/mustard_easy_00_02/masks/`

The method saves the init frame correctly:
![1581268472043767456](https://user-images.githubusercontent.com/30011340/228888512-a3262997-1ea1-46f8-911c-1e0e794bda90.png)
And then the second frame looks odd:
![1581268472102247284](https://user-images.githubusercontent.com/30011340/228888554-7dd81621-a07f-4452-bcf2-3a3fa0a9db5c.png)

And every frame after this is even worse. This is the last one:
![1581268483826868698](https://user-images.githubusercontent.com/30011340/228889234-7689d2a1-9eb6-4d86-a211-b491c83d8d47.png)

The code was not touched except for changing:
```python
 input = rgb_normalize(input).unsqueeze(0)
```
to 
```python
input = torch.Tensor(input/255.).permute(2,0,1)
input = transforms.Normalize(mean=[0.485, 0.456, 0.406],std=[0.229, 0.224, 0.225]) (input)
input = input.unsqueeze(0)
```
since for some reason the transforms.ToTensor() transformation was expecting a np.ndarray but receiving a numpy.ndarray (which should be interchangable, but for some reason was throwing an error?).

The weights I got from the download link and put in BundleTrack/transductive-vos.pytorch/pretrained.

Am I doing something wrong?

**Comment by dennisushi** [comment]
Using the [3090 docker](https://hub.docker.com/r/wenbowen123/bundletrack/tags) fixed the issue.
